docs/how-to-guides/upgrading-renku.rst

```diff
@@ -28,60 +28,28 @@ Upgrading your image to use the latest ``renku`` CLI version
-
-2. Push your changes (either from the GitLab editor, or in a running Session), 
-   and start up a new Session from the latest commit to use this new image.
-
-3. When you start to run ``renku`` commands, you might be asked to call ``renku migrate``.
-   This command ensures that the metadata in your project is updated.
-
-Test drive
-^^^^^^^^^^
-
-If you just want to try out the latest version of ``renku`` without building a new
-image yet, you can follow the ``pipx`` installation instructions from
-:ref:`upgrading your local installation <upgrading_local>`. Installing in this way,
-like any other terminal install, will not hold through the next time you launch.
-
-Keep in mind that your project's metadata will be upgraded when you start running
-the new version ``renku`` commands - don't push these changes if you want to revert
-back to an older version.
-
-You should follow the steps in the previous section to swap out the base image
-once you're satisfied with latest ``renku``.
-
-Only updating the renku CLI
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-If you want to use the latest version of ``renku`` but for some reason cannot rebuild
-from scratch the rest of the docker image layers for your project (e.g. dependency issues),
-you can add the ``pipx`` installation from
-:ref:`upgrading your local installation <upgrading_local>` to the end of your ``Dockerfile``;
-e.g. ``RUN pipx install renku==<VERSION> --force``.
-
-When the gitlab CI builds an image using the ``Dockerfile`` in your project, it
-re-uses layers of installs that haven't changed between builds, which speeds up
-the build process. But, when you swap out the base image (as is recommended), all
-of the subsequent layers might need to rebuild.
-
-We still recommend swapping out the base image when possible.
+``ARG RENKU_VERSION`` line of the ``Dockerfile`` in your project repo (if you 
+don't see this line in your ``Dockerfile`` your project was made using an older template).
+All you need to do to have a different version of the Renku CLI installed is modify the version 
+on that line and push the change. Once that is done, the image will automatically
+be built as usual and include the new version. 
+
+.. _renku_base_image_upgrade:
+
+Upgrading the base image used for interactive sessions
+------------------------------------------------------
+
+In addition to updating the CLI version, you may also want to update the base image 
+used for sessions in your project. We periodically release new base images with 
+upgrades to underlying libraries and packages. The base image is specified 
+on the ``ARG RENKU_BASE_IMAGE`` line in your ``Dockerfile``, for example it might
+read ``renku/renkulab-py:3.9-0.11.0``. This means that it is a python-based image 
+using Python 3.9 and image release ``0.11.0``. To change it, simply modify the base image
+referenced on that line. For a list of base images see
+`current images <https://github.com/SwissDataScienceCenter/renkulab-docker#current-images>`_.
```

Comment:
This link also has base images like [python-3.7.6-14f93c5](https://hub.docker.com/layers/renku/renkulab-py/python-3.7.6-14f93c5/images/sha256-0b6ac6a477d7b2cab40037bed4d4873a80b26a49b95b87dd1f72ef958df520d6?context=explore) but I don't think we want users to upgrade to this. I wonder if we should point this out just to prevent a user using a dev image?

Reply:
you mean if you follow the links to dockerhub? That's a good point - I could say that people should generally use released images and link to the releases page?

Reply:
I updated the wording.